examples/3d/3d_shapes.rs

```diff
@@ -84,10 +85,14 @@ fn setup(
         ..default()
     });
 
-    commands.spawn(Camera3dBundle {
-        transform: Transform::from_xyz(0.0, 6., 12.0).looking_at(Vec3::new(0., 1., 0.), Vec3::Y),
-        ..default()
-    });
+    commands
+        .spawn(Camera3dBundle {
+            transform: Transform::from_xyz(0.0, 6., 12.0)
+                .looking_at(Vec3::new(0., 1., 0.), Vec3::Y),
+            ..default()
+        })
+        .insert(NoCpuCulling)
+        .insert(GpuCulling);
```

Comment:
I'm not sure about these additions to the `3d_shapes`, `lighting`, and `deferred_rendering`. Maybe a new example is in order to test the culling combinations specifically?

Reply:
Oh, yeah, I missed that. I think it should just be removed.

Reply:
Oh, oops, I meant to remove those but forgot :)

Reply:
Removed.